naga/src/back/pipeline_constants.rs

```diff
@@ -14,7 +14,10 @@ use thiserror::Error;
 pub enum PipelineConstantError {
     #[error("Missing value for pipeline-overridable constant with identifier string: '{0}'")]
     MissingValue(String),
-    #[error("Source f64 value needs to be finite (NaNs and Inifinites are not allowed) for number destinations")]
+    #[error(
+        "Source f64 value needs to be finite ({}) for number destinations",
+        "NaNs and Inifinites are not allowed"
+    )]
```

Comment:
I'm sure there's a reason, but, explain to me why we can't just do

```rust
#[error(
    "Source f64 value needs to be finite \
     (NaNs and Infinities are not allowed) \
     for number destinations"
)]
```

Reply:
It will (marginally) help debugging if you say what the value is so you know what to look for.

Reply:
The reason is that I simply hadn't considered it, TBH. I susepct it might still break `rustfmt`, but worth a try!

Reply:
> It will (marginally) help debugging if you say what the value is so you know what to look for.

@cwfitzgerald "Please File An Issue"

Reply:
Confirmed that this breaks `rustfmt`'s indentation, though not surrounding expressions. The literal will simply never have the indentation of the second and subsequent lines after the first line break.

I also tried `"{}", concat!(…)`, but this also breaks `rustfmt`. 😩